docker/cache/resources/build-cache.sh

```diff
@@ -49,6 +49,10 @@ if [ -n "${FORCE_REFRESH}" ]; then
         curl -X POST ${STATEMENTS_ENDPOINT} -H "Content-Type: application/x-www-form-urlencoded" -H "Accept: application/sparql-results+json" --data-urlencode "update@$filename"
     done
 
+    #sleep for a bit, just to make sure last command has flushed
+    sleep 30
```

Comment:
Have we established that the problem is definitely that the GRAPHDB returns a 200 before it has actually completed INSERT? If so it might be worth having, in addition to this wait a 999_Null.sparql script that inserts something small into the cache to future proof in case we add a pre conditioning script at the last step that takes longer than 30 minutes to run. If that makes sense it can either either happen as part of this pull request or go in as bug?

Reply:
Agreed, this is simple a first crack at helping the problem. Can spend more time investigating at a later stage.